src/shared/modules/dbMeta/dbMetaEpics.ts

```diff
@@ -546,6 +547,12 @@ export const serverConfigEpic = (some$: any, store: any) =>
             store.dispatch(triggerCredentialsTimeout())
           }
 
+          setTimeout(() => {
```

Comment:
interesting that it works even with a timeout of `0`

Reply:
indeed - it's where I start to get a bit iffy with the eventloop/callback queue - but I guess the callback just going to the back of the queue is enough for react's internals to finish and resolve props in the app